### DISCUSSION TITLE
GFX Card support

### DISCUSSION THREAD

**Discussion by PsyaNydeUK** [ixjf/MSIRGB#185] [Ideas]
Would be great if this also worked for the MSI GFX card LED's as well, unfortunately at the moment you have to install the dreadful MSi Dragon center in order to get the gfx cards to work in mystic light

**Comment by ixjf** [comment]
Not in my plans. You can check out OpenRGB, check if it supports these.

**Reply by PsyaNydeUK** [reply]
wow, works great for my MSi RTX2080 Super X-Trio, thanks :) Finally, the 'rainbow' effect has been conquered.
